include/json/features.h

```diff
@@ -52,6 +52,9 @@ class JSON_API Features {
 
   /// \c true if numeric object key are allowed. Default: \c false.
   bool allowNumericKeys_;
+
+  /// \c true if duplicate keys within an object should be rejected. Default: \c false.
+  bool rejectDupKeys_;
```

Comment:
Oh, you like the exposed error-offsets. I didn't think that was used for anything except jsoncpp exceptions.

Well, you might be better off with your own fork, unfortunately.